docs/source/tutorials/datalab/image.ipynb

```diff
@@ -173,7 +173,7 @@
     "\n",
     "# Apply transformations\n",
     "def normalize(example):\n",
-    "    example[\"image\"] = (example[\"image\"] / 255.0).unsqueeze(0)\n",
+    "    example[\"image\"] = (example[\"image\"] / 255.0)\n",
```

Comment:
does this change mean the code no longer works on older versions of the `datasets` package? If so would be good to specify which is the lower-bound version at the top of tutorial

Reply:
That's right.
It's an backwards incompatible change in the datasets package that affects the dataset.with_format("torch") call